src/main/java/org/aludratest/service/flatfile/impl/DefaultFlatFileAction.java

```diff
@@ -192,6 +192,16 @@ public String toString() {
         }
 
         @Override
+        public boolean equals(Object obj) {
+            if (this == obj)
+                return true;
+            if (obj == null || getClass() != obj.getClass())
+                return false;
```

Comment:
Please use curly braces for both if blocks although they are one-liners, as this is our style... Thanks!